File: Samples/Quiz_WindowsPhone/QuizMania/QuizMania/QuizModels/QuestionPacket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace QuizMania.QuizModels
{
  public class QuestionPacket
    {
        public int Id;
        public QuizQuestion Question;
        public List<QuizElement> Options;

        public QuestionPacket(int Id, QuizQuestion Question, List<QuizElement> Options)
        {
            this.Id = Id;
            this.Question = Question;
            this.Options = Options;
        }

        public int getId()
        {
            return Id;
        }

        public void setId(int Id)
        {
            this.Id = Id;
        }

        public QuizQuestion getQuestion()
        {
            return Question;
        }

        public void setQuestion(QuizQuestion Question)
        {
            this.Question = Question;
        }

        public List<QuizElement> getOptions()
        {
            return Options;
        }

        public void setOptions(List<QuizElement> Options)
        {
            this.Options = Options;
        }
    }
}

File: Samples/Quiz/QuizMania/src/com/appwarp/s2/util/QuestionPacket.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.appwarp.s2.util;

import java.util.ArrayList;

/**
 *
 * @author rahulwindows
 */
public class QuestionPacket {

    public int Id;
    public QuizQuestion Question;
    public ArrayList<QuizElement> Options;

    public QuestionPacket(int Id, QuizQuestion Question, ArrayList<QuizElement> Options) {
        this.Id = Id;
        this.Question = Question;
        this.Options = Options;
    }

    public int getId() {
        return Id;
    }

    public void setId(int Id) {
        this.Id = Id;
    }

    public QuizQuestion getQuestion() {
        return Question;
    }

    public void setQuestion(QuizQuestion Question) {
        this.Question = Question;
    }

    public ArrayList<QuizElement> getOptions() {
        return Options;
    }

    public void setOptions(ArrayList<QuizElement> Options) {
        this.Options = Options;
    }
}
